bitnami/kibana/templates/configmap.yaml

```
@@ -26,9 +26,9 @@ data:
     server.publicBaseUrl: {{ include "common.tplvalues.render" ( dict "value" .Values.configuration.server.publicBaseUrl "context" $ ) }}
     {{- else }}
     {{- if .Values.ingress.enabled }}
-    {{- $protocol := ternary "https" "http" (or .Values.ingress.tls .Values.ingress.extraTls) -}}
-    {{- $hostname := required "Ingress hostname must be specified" .Values.ingress.hostname -}}
-    server.publicBaseUrl: {{ printf "%s://%s" $protocol ( include "common.tplvalues.render" ( dict "value" $hostname ) "context" $ ) }}
+    {{- $protocol := ternary "https" "http" .Values.ingress.tls -}}
```

The last `-` can be removed also
```suggestion
    {{- $protocol := ternary "https" "http" .Values.ingress.tls }}
```